src/main/java/mat/server/MeasureLibraryServiceImpl.java

```diff
@@ -5931,8 +5936,7 @@ public String getHumanReadableForMeasureDetails(String measureId) {
             standardizeStartAndEndDate(measureInformationModel);
             humanReadableHTML = humanReadableGenerator.generate(measureInformationModel);
         } catch (Exception e) {
-            e.printStackTrace();
-            logger.info("Exception in getHumanReadableForMeasureDetails: " + e);
+            logger.info("Exception in getHumanReadableForMeasureDetails: " + e, e);
```

Comment:
Shouldn't this be `logger.error("msg", e)?`

Reply:
oh, good catch